tests/functional/ui/test_error_improvements.py

```diff
@@ -112,6 +114,7 @@ def test_namespace_store_creation_rules(
     @bugzilla("2193109")
     @polarion_id("OCS-4873")
     @skipif_hci_provider_or_client
+    @skipif_disconnected_cluster
```

Comment:
It doesn't look like `test_blocking_pool_creation_rules` is reaching the outside network - please remove this skip decorator.

Reply:
you're absolutely right 💯 
removed